modules/external-api/src/main/java/org/opencastproject/external/endpoint/EventsEndpoint.java

```diff
@@ -1641,8 +1642,8 @@ public Response getEventPublications(@HeaderParam("Accept") String acceptHeader,
 
   private List<JValue> getPublications(Event event, Boolean withSignedUrls, Boolean includeInternalPublication, ApiVersion requestedVersion) {
         return event.getPublications().stream()
-        .filter(publication -> ((includeInternalPublication && !requestedVersion.isSmallerThan(VERSION_1_11_0)) || EventUtils.internalChannelFilter.apply(publication)))
-        .map(p -> getPublication(p, withSignedUrls, requestedVersion))
+            .filter(publication -> ((includeInternalPublication != null && includeInternalPublication && !requestedVersion.isSmallerThan(VERSION_1_10_0)) || EventUtils.internalChannelFilter.apply(publication)))
```

Comment:
The version is off, most likely a copy-paste error from the client repo where 1.11 doesn't exist yet. :sweat_smile: